File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        if (n == 0) return 0;
        
        int left = 0;
        vector<int> New;
        New.push_back(nums[left]); 
        
        for (int right = 1; right < n; right++) {
            if (nums[right] != nums[left]) {
                left = right;
                New.push_back(nums[left]);
            }
        }
        for (int i = 0; i < New.size(); i++) {
            nums[i] = New[i];
        }
        return New.size();
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        no = Counter(nums)
        unique_keys = sorted(no.keys())
        for i in range(len(nums)):
           if i < len(unique_keys):
             nums[i] =  unique_keys[i]
           else:
             nums.pop()
        return len(unique_keys)
    
       
        